admin-dev/themes/new-theme/scss/components/_tinymce.scss

```diff
@@ -1,1215 +1,1517 @@
+      background-color: #d6d6d6;
+    }
+  }
+  &:not(.mce-disabled):active {
+    background-color: #d6d6d6;
+  }
+  button {
+    padding: 2px 6px;
+    overflow: visible;
+    font-size: 12px;
+    color: #333;
+    text-align: center;
+    cursor: pointer;
+
+    &::-moz-focus-inner {
+      padding: 0;
+      border: 0;
+    }
+  }
+}
 
 .mce-primary {
-    min-width: 50px;
+  min-width: 50px;
+  color: #fff;
+  background-color: #006dcc;
+  border: 1px solid #b1b1b1;
+  border-color: rgba(0, 0, 0, .1) rgba(0, 0, 0, .1) rgba(0, 0, 0, .25) rgba(0, 0, 0, .25);
+  &:hover,
+  &:focus {
+    background-color: #005fb3;
+  }
+  &.mce-disabled {
+    button,
+    &:hover button {
+      cursor: default;
+      zoom: 1;
+      filter: alpha(opacity = 40);
+      opacity: .4;
+    }
+  }
+  &.mce-active {
+    background-color: #005299;
+    &:hover {
+      background-color: #005299;
+    }
+  }
+  &:not(.mce-disabled):active {
+    background-color: #005299;
+  }
+  button {
     color: #fff;
-    border: 1px solid #b1b1b1;
-    border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25) rgba(0, 0, 0, 0.25);
-    background-color: #006dcc;
-    &:hover, &:focus {
-        background-color: #005fb3; }
-    &.mce-disabled {
-        button, &:hover button {
-            cursor: default;
-            opacity: .4;
-            filter: alpha(opacity = 40);
-            zoom: 1; } }
-    &.mce-active {
-        background-color: #005299;
-        &:hover {
-            background-color: #005299; } }
-    &:not(.mce-disabled):active {
-        background-color: #005299; }
-    button {
-        color: #fff;
-        i {
-            color: #fff; } } }
-
-.mce-btn-large {
-    button {
-        padding: 9px 14px;
-        font-size: 12px;
-        line-height: normal;
-        border-radius: 5px; }
     i {
-        margin-top: 2px; } }
+      color: #fff;
+    }
+  }
+}
+
+.mce-btn-large {
+  button {
+    padding: 9px 14px;
+    font-size: 12px;
+    line-height: normal;
+    @include border-radius(5px);
+  }
+  i {
+    margin-top: 2px;
+  }
+}
 
 .mce-btn-small {
-    button {
-        padding: 1px 5px;
-        font-size: 12px;
-        *padding-bottom: 2px; }
-    i {
-        line-height: 20px;
-        vertical-align: top;
-        *line-height: 18px; } }
+  button {
+    padding: 1px 5px;
+    *padding-bottom: 2px;
+    font-size: 12px;
+  }
+  i {
+    line-height: 20px;
+    *line-height: 18px;
+    vertical-align: top;
+  }
+}
 
-.mce-btn .mce-caret, .mce-btn-small .mce-caret {
-    margin-top: 8px;
-    margin-left: 0; }
+.mce-btn .mce-caret,
+.mce-btn-small .mce-caret {
+  margin-top: 8px;
+  margin-left: 0;
+}
 
 .mce-caret {
-    display: inline-block;
-    *display: inline;
-    *zoom: 1;
-    width: 0;
-    height: 0;
-    vertical-align: top;
-    border-top: 4px solid #333;
-    border-right: 4px solid transparent;
-    border-left: 4px solid transparent;
-    content: ""; }
+  display: inline-block;
+  *display: inline;
```

Comment:
IE hack?

Reply:
I don't know at all the hell is that and I doubt that's from a modification I did, maybe stylelint.. But I can't find anything related to *display, that's weird

Reply:
It was already present it's not you nor eslint It's a CSS hack to target IE7 and below http://www.javascriptkit.com/dhtmltutors/csshacks3.shtml

Reply:
Alright then